modules/ztp-configuring-a-static-ip.adoc

```diff
@@ -35,34 +35,34 @@ spec:
      - name: eth0
        type: ethernet
        state: up
-       mac-address: <mac-address> <1>
        ipv4:
          enabled: true
          address:
-           - ip: <ip-address> <2>
-             prefix-length: <public-network-prefix> <3>
+           - ip: <ip-address> <1>
+             prefix-length: <public-network-prefix> <2>
          dhcp: false
    dns-resolver:
      config:
        server:
-         - <dns-resolver> <4>
+         - <dns-resolver> <3>
    routes:
      config:
        - destination: 0.0.0.0/0
-         next-hop-address: <gateway> <5>
+         next-hop-address: <gateway> <4>
          next-hop-interface: eth0
          table-id: 254
  interfaces:
-   - name: "eth0" <6>
-     macAddress: <mac-address> <7>
+   - name: "eth0" <5>
+     macAddress: <mac-address> <6>
 ----
-<1> `mac-address` is the MAC address of the target bare metal machine, that is, the same MAC address used in the `BareMetalHost` resource.
-<2> `ip-address` is the static IP address of the target bare metal machine.
-<3> `public-network-prefix` is the static IP address’s subnet for the target bare metal machine.
-<4> `dns-resolver` is the DNS server for the target bare metal machine.
-<5> `gateway` is the gateway for the target bare metal machine.
-<6> `name` must match the name specified in the `interfaces` section.
-<7> `mac-address` must match the MAC address specified in the `interfaces` section.
+<1> `ip-address` is the static IP address of the target bare metal machine.
```

Comment:
According to the ISG, user-defined variables should be enclosed in brackets, even when they are reffered to in the callouts. Example: [The callouts in step two of the procedure.](https://docs.openshift.com/container-platform/4.9/installing/installing-mirroring-installation-images.html#installation-adding-registry-pull-secret_installing-mirroring-installation-images)

Since the rest of the assembly does not follow this style convention, perhaps it makes sense to fix this throughout the book as a continuous improvement iteam. Alternatively, I think that most (if not all) of these variables do not need to be repeated in the callouts. For example, "<1> The static IP address of the target bare metal machine," is very clear and direct.